src/content/contributing/community-membership/_index.en.md

```diff
@@ -53,6 +57,7 @@ are expected to remain active contributors to the community.
 ### Requirements
 
 - Enabled [two-factor authentication] on their GitHub account
+- Compliance with [Developer Certificate Of Origin (DCO)](https://developercertificate.org/). This can be done by adding --signoff to your Git command line.
```

Comment:
This shouldn’t be listed as a membership requirement: it’s a requirement on each individual commit, not on contributors.